src/SamplesApp/SamplesApp.netcoremobile/SamplesApp.netcoremobile.csproj

```diff
@@ -50,13 +52,6 @@
 
 		<When Condition="'$(TargetPlatformIdentifier)'=='ios'">
 			<PropertyGroup Condition="'$(TargetPlatformIdentifier)'=='ios'">
```

Comment:
^  not a change from this PR but `Condition` is duplicated on both `When` and `PropertyGroup`